### DISCUSSION TITLE
3.6.0

### DISCUSSION THREAD

**Discussion by HFO4** [cloudreve/cloudreve#1560] [General]
[中文图文介绍](https://forum.cloudreve.org/d/2651)

### 🎇Highlights

This version already includes all changes from [3.6.0-beta1](https://github.com/cloudreve/Cloudreve/releases/tag/3.6.0-beta1), which will not br covered here.

### 🎁 New Features

* [Pro] Purchase volume licensing of Cloudreve iOS App.
* Promotion page for connecting to mobile App.
* Set `max-age` for publicly accessible resources (e.g. source links, download links).
* Customize socket file permission (#749).

### 🛠 Enhancement

* Show error message in tooltip if transferring task failed in aria2 list.
* Instruction for creating AAD application which allows consumer Microsoft account to login (#1442).

### 🐛 Bug Fixes

* [Pro] Only node with "remote download" feature enabled can be selected by users (#1534, #1525).
* Minor typos and missed translations

<hr /><em>This discussion was created from the release <a href='https://github.com/cloudreve/Cloudreve/releases/tag/3.6.0'>3.6.0</a>.</em>

**Comment by hzxie** [comment]
Failed to start after upgrading to 3.6.0.

```
[Info]    2022-12-15 09:33:56 Initializing database connection...
[Info]    2022-12-15 09:33:56 Start initializing database schema...
[Panic]   2022-12-15 09:33:56 Failed to create initial root user: Error 1062: Duplicate entry 'admin@cloudreve.org' for key 'uix_cd_users_email'
panic: Failed to create initial root user: Error 1062: Duplicate entry 'admin@cloudreve.org' for key 'uix_cd_users_email'

goroutine 1 [running]:
github.com/cloudreve/Cloudreve/v3/pkg/util.(*Logger).Panic(0xc000422d20, {0x131073a?, 0xc00098e380?}, {0xc0003039b8?, 0x0?, 0x0?})
        /home/runner/work/Cloudreve/Cloudreve/pkg/util/logger.go:74 +0x94
github.com/cloudreve/Cloudreve/v3/models.addDefaultUser()
        /home/runner/work/Cloudreve/Cloudreve/models/migration.go:177 +0x39f
github.com/cloudreve/Cloudreve/v3/models.migration()
        /home/runner/work/Cloudreve/Cloudreve/models/migration.go:53 +0x3cf
github.com/cloudreve/Cloudreve/v3/models.Init()
        /home/runner/work/Cloudreve/Cloudreve/models/init.go:89 +0x7ca
github.com/cloudreve/Cloudreve/v3/bootstrap.Init.func3()
        /home/runner/work/Cloudreve/Cloudreve/bootstrap/init.go:47 +0x17
github.com/cloudreve/Cloudreve/v3/bootstrap.Init({0xc000040480, 0x2b}, {0x17d43c0, 0xc00000e588})
        /home/runner/work/Cloudreve/Cloudreve/bootstrap/init.go:102 +0x254
main.init.0()
        /home/runner/work/Cloudreve/Cloudreve/main.go:40 +0x1ca
```

**Reply by hzxie** [reply]
The issue can be resolved by deleting the user "admin@cloudreve.org" manually in the database.
Where does the user "admin@cloudreve.org" come from? I never created it.

**Reply by hzxie** [reply]
I also got a warning message saying: `[Warning] 2022-12-15 09:39:00 Static resource version mismatch [Current 3.6.0-beta1, Desired: 3.6.0]，please delete "statics" folder and rebuild it.`

**Comment by homestoo** [comment]
大佬什么时候有空可以更新下docker的最新版本吗
